Formula/geoserver.rb

```diff
@@ -25,6 +29,6 @@ def caveats; <<~EOS
   end
 
   test do
-    assert_match "geoserver path", shell_output("#{bin}/geoserver")
+    assert_match "Usage: $ geoserver -start | -stop | path/to/data/dir", shell_output("#{bin}/geoserver")
```

Comment:
Is there a way to add a better test here?

Testing the usage output in the script that the formula just wrote is rather tautological, and does nothing to check the functionality of the actual software that is being installed.